### DISCUSSION TITLE
Does nicegui have any examples of large websites?

### DISCUSSION THREAD

**Discussion by ponponon** [zauberzeug/nicegui#2085] [Q&A]
### Question

Does nicegui have any examples of large websites?

The biggest site I've seen so far is nicegui's website, but nicegui's website is more like a static website.

Of course, there are a lot of examples on the website, but they are just simple demos.

I would like to know if nicegui has more examples of large websites. I would like to learn and study them.

**Comment by stdusr** [comment]
Would be really interested, also. But I guess, the people who created "large websites" are quite busy right now :) Personally, my "largest" project is a crypto arbitrage bot with a frontend built with NiceGUI:

![368566036_873813401078027_642925114315690536_n](https://github.com/zauberzeug/nicegui/assets/4020805/63c2b1cf-db85-45bd-a1bd-a706a3ccf587)

It has quite a bit of functionality under the hood, but, as an amateur coder, I spent 70% of my time fighting with layout and positioning, though it looks fairly simple :)

**Reply by natankeddem** [reply]
Happy to see people making use of `json_editor` I wasn't sure if I was the only one with use for it. 🤣

**Reply by stdusr** [reply]
That json_editor is a life saviour when debugging and investigating super deeply nested JSON structures :)

**Comment by falkoschindler** [comment]
There's also a collection of NiceGUI projects on our [wiki](https://github.com/zauberzeug/nicegui/wiki/#community-projects). Not all of them are particularly large, but especially with [RoSys](https://github.com/zauberzeug/rosys/) we've already built quite complex robotics applications. Unfortunately they are usually closed-source.
